modules/workflow-workflowoperation/src/main/java/org/opencastproject/workflow/handler/workflow/DuplicateEventWorkflowOperationHandler.java

```diff
@@ -155,6 +205,9 @@ public WorkflowOperationResult start(final WorkflowInstance workflowInstance, fi
     final String configuredSourceFlavors = trimToEmpty(operation.getConfiguration(SOURCE_FLAVORS_PROPERTY));
     final String configuredSourceTags = trimToEmpty(operation.getConfiguration(SOURCE_TAGS_PROPERTY));
     final String configuredTargetTags = trimToEmpty(operation.getConfiguration(TARGET_TAGS_PROPERTY));
+    final boolean noSuffix = Boolean.parseBoolean(trimToEmpty(operation.getConfiguration(NO_SUFFIX)));
+    final String seriesIdStr = operation.getConfiguration(SET_SERIES_ID);
+    final String seriesId = seriesIdStr == null || seriesIdStr.startsWith("$") ? "" : seriesIdStr;
```

Comment:
I'm curious why the set-series-id value in the operation configuration might start with a "$"? It seems like a really odd edge case.

Reply:
Good catch. I remember this code making sense when I set up the new WOH, but I've now tested it without this code and it still works (I also removed parameters from the `duplicate-event` workflow).